File: core/src/org/rexcellentgames/burningknight/debug/DebugCommand.java
package org.rexcellentgames.burningknight.debug;

import org.rexcellentgames.burningknight.physics.World;

public class DebugCommand extends ConsoleCommand {
	{
		name = "/debug";
		shortName = "/d";
	}

	@Override
	public void run(Console console, String[] args) {
		World.DRAW_DEBUG = !World.DRAW_DEBUG;
	}
}

File: core/src/org/rexcellentgames/burningknight/debug/DebugCommand.kt
package org.rexcellentgames.burningknight.debug

import org.rexcellentgames.burningknight.physics.World

class DebugCommand : ConsoleCommand("/debug", "/d", "Toggles physical debug") {
	override fun run(console: Console, args: Array<String>) {
		World.DRAW_DEBUG = !World.DRAW_DEBUG
	}
}